cdap-docs/developers-manual/source/building-blocks/datasets/fileset.rst

```diff
@@ -317,6 +317,58 @@ For example, give these arguments when starting the MapReduce through a RESTful
     "dataset.totals.output.partition.key.league" : "nfl"
   }
 
+Incrementally Processing Partitions of a PartitionedFileSets in MapReduce
```

Comment:
Incrementally Processing Partitions of a PartitionedFileSets

The title is otherwise too long, and as the lead says, this is just one way to process a fileset...